extensions/winrt/_app_service/AppServiceClosedEventArgs.cpp

```diff
@@ -0,0 +1,112 @@
+#include "pch.h"
+#include "event.h"
+
+using namespace Microsoft::WRL;
+using namespace Windows::Foundation;
+using namespace Windows::ApplicationModel::AppService;
+
+namespace PyAppServiceClosedEventArgs
+{
+    typedef AppServiceClosedEventArgs thisType;
+#pragma region getset
+
+    static PyObject * getStatus(PyObject *obj, void *closure)
+    {
+        PyObject* const FailureValue = NULL;
+        return ExceptionGuard([=]
+        {
+            PyWinRTObject *self = reinterpret_cast<PyWinRTObject*>(obj);
+            auto args = Platform_cast<thisType>(self->pInspectable);
+            auto appServiceClosedStatus_type = GetTypeFromModule("WinRT.ApplicationModel.AppService", "AppServiceClosedStatus");
+            return PyObject_CallFunction((PyObject*)appServiceClosedStatus_type, "i", static_cast<int>(args->Status));
+        }, FailureValue);
+    }
+
+    static PyGetSetDef getset[] = {
+        { "Status", (getter)getStatus, NULL, "Status -> AppServiceClosedStatus. \nGets the status that was set when the endpoint for the app service was closed.", NULL },
+        { NULL }  /* Sentinel */
+    };
+
+#pragma endregion getset
+
+#pragma region methods
+
+    static PyMethodDef methods[] = {
+        { "_CreateFromIInspectable", WinRTHelper<thisType>::CreateFromInspectable, METH_CLASS | METH_VARARGS, NULL },
+        { NULL, NULL },
+    };
+
+#pragma endregion methods
+
+    static const char type_name[] = "WinRT.ApplicationModel.AppService.AppServiceClosedEventArgs";
+    static const char docstring[] = "Provides data for the AppServiceConnection.ServiceClosed event that occurs when the other endpoint closes connection to the app service.";
+
+    PyTypeObject Type = {
```

Comment:
No.  You need a different instance of PyTypeObject for each python type that you are defining.  See https://docs.python.org/3/extending/newtypes.html.